tests/system/db_adapter/base_relational_db_test.py

```diff
@@ -0,0 +1,49 @@
+import os
+from typing import TypedDict
+from unittest import TestCase
+
+from psycopg2 import connect
```

Comment:
IMO using psycopg2 and psycopg(3) in parallel is no good idea

Reply:
I agree, but for the transition, it is easiest to keep both. As more and more datstore code rotates out, we can update the code to version 3